packages/components/tree-v2/src/composables/useTree.ts

```diff
@@ -263,6 +263,11 @@ export function useTree(props: TreeProps, emit) {
     nextTick(() => (tree.value = createTree(data)))
   }
 
+  function getNode(data: TreeKey | TreeNodeData) {
+    const key = typeof data !== 'object' ? data : getKey(data)
```

Comment:
```suggestion
    const key = isObject(data) ? data : getKey(data)
```

You can use `isObject` from `@element-plus/utils`

Reply:
> You can use `isObject` from `@element-plus/utils`

It looks like it should be `!isObject(data) ? data : getKey(data)`